indexing-service/src/main/java/io/druid/indexing/common/task/TaskUtils.java

```diff
@@ -26,8 +26,7 @@
 {
   public static String makeId(String id, final String typeName, String dataSource, Interval interval)
```

Comment:
I thought about removing it, but it does provide some standard structure over the general joiner. Meaning that it is expected that an ID will be in the `type_datasource_start_end_timestamp` format, while still retaining the arbitrary capacity of AbstractTask.joinId